.gitlab-ci.yml

```diff
@@ -47,10 +73,40 @@ build:docker:ui:
   variables:
     DOCKER_REPOSITORY: mendersoftware/mantra-ui
     DOCKER_DIR: ui
+  after_script:
+    - cp image.tar image-ui.tar
+  artifacts:
+    expire_in: 2w
+    paths:
+      - image.tar
+      - image-ui.tar
   rules:
     - changes:
         - ui/**/*
 
+test:composition:
+  stage: acceptance_tests
+  image: tiangolo/docker-with-compose
+  needs:
+    - job: build:docker
+      artifacts: true
+    - job: build:docker:ui
+      artifacts: true
+  services:
+    - docker:20.10.8-dind
```

Comment:
S:
Why not `20.10`, so at least we get patches? :smile:

Reply:
errr... 🤷‍♂️ just to keep it in line with other repos as we know this version works (since there have been some that didn't play nice with the gitlab CI)